packages/mongodb-chatbot-ui/src/conversationStore.ts

```diff
@@ -49,12 +49,28 @@ export type ConversationStore = ReturnType<typeof makeConversationStore>;
 
 export const makeConversationStore = (name = "default") => {
   return createStore(
-    combine(initialConversationState, (set) => ({
+    combine(initialConversationState, (set, get) => ({
       name,
       api: {
         initialize: (initialState: ConversationState) => {
           set(initialState);
         },
+        setConversationId: (conversationId: string) => {
+          if (conversationId === "") {
+            console.warn("Attempted to set an empty conversation ID");
```

Comment:
should these console.warn() calls be turned off for prod?

Reply:
Probably doesn't matter that much, but for consistency with other warnings I'll update